Userland/Utilities/uniq.cpp

```diff
@@ -5,75 +5,122 @@
  */
 
 #include <AK/RefPtr.h>
+#include <AK/StringView.h>
 #include <LibCore/ArgsParser.h>
+#include <LibCore/Stream.h>
 #include <LibCore/System.h>
-#include <errno.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
 #include <unistd.h>
 
-struct linebuf {
-    char* buf = NULL;
-    size_t len = 0;
-};
-
-static FILE* get_stream(char const* filepath, char const* perms)
+static ErrorOr<void> write_line_content(StringView line, size_t count, bool show_duplicates, bool print_count, Core::Stream::File& outfile)
```

Comment:
The `show_duplicates` here should be `show_only_duplicates`, right?

Reply:
true, the variable name is different at the bottom, it's `duplicates_only`. ill change them so that they're all `duplicates_only`